tests/app/organisation/test_rest.py

```diff
@@ -85,6 +85,41 @@ def test_get_organisation_by_id_returns_domains(admin_request, notify_db_session
     }
 
 
+@pytest.mark.parametrize('domain, expected_status', (
```

Comment:
You might consider a test when no organisation is found.

Reply:
That’s the `404` I’m testing here:
https://github.com/alphagov/notifications-api/pull/2434/files#diff-3525e2f6804d98e0a7cbf330f454721cR91